packages/ui-components/src/components/Dialog.js

```diff
@@ -30,7 +30,7 @@ export const Dialog = ({ children, ...props }) => (
 );
 
 Dialog.propTypes = {
-  children: PropTypes.array.isRequired,
+  children: PropTypes.any.isRequired,
```

Comment:
Children can be a lot of things. Array, Object, Function, Node, Null, Undefined etc